integration-tests/contract-originate-contract-with-sapling.spec.ts

```diff
@@ -1,23 +1,23 @@
 import { CONFIGS } from './config';
-import { BigNumber } from 'bignumber.js';
-import { RpcClient, RpcClientCache } from '@taquito/rpc';
 import { Protocols } from '@taquito/taquito';
-import { SaplingStateValue } from '../packages/taquito-michelson-encoder/src/taquito-michelson-encoder';
 import { SaplingStateAbstraction } from 'taquito/src/contract/sapling-state-abstraction';
-import { saplingContract } from './data/sapling_contracts';
 import {
   saplingContractDouble,
   saplingContractDrop,
   saplingContractSend,
   saplingContractStateAsArg,
   saplingContractPushSaplingState,
   saplingContractUseExistingState,
+  saplingContractDoubleWithJProtoFix,
```

Comment:
Using the 'Fix' postfix might imply that the values are somewhat temporary. Maybe we could remove the 'Fix' postfix since we will probably use these values in the long term.

Reply:
thanks for the suggestion, I renamed the contracts